core_impl/src/main/java/io/opencensus/stats/MeasureToViewMap.java

```diff
@@ -84,47 +84,52 @@ synchronized void registerView(View view, Clock clock) {
   }
 
   // Records stats with a set of tags.
-  synchronized void record(StatsContextImpl tags, MeasureMap stats) {
+  synchronized void record(StatsContextImpl tags, MeasureMap stats, Clock clock) {
```

Comment:
I've updated `MeasureToViewMap.record()` and `MutableViewData.record()` to take `Timestamp` instead of `Clock`.